ambari-server/src/main/resources/stacks/BIGTOP/3.2.0/services/HDFS/package/scripts/service_check.py

```diff
@@ -81,12 +81,13 @@ def service_check(self, env):
     params.HdfsResource(None, action="execute")
 
     if params.has_journalnode_hosts:
+      journalnode_web_path = "/journalnode.html"
```

Comment:
Why we use `journalnode.html` instead of `index.html` here?

Reply:
It's also okay if using index.html but in fact index.html will jump to journalnode.html